shared/rSharp.props

```diff
@@ -0,0 +1,15 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ImportGroup Label="PropertySheets" />
+  <PropertyGroup Label="UserMacros">
+    <RInstallPath>C:\PROGRA~1\R\R-4.3.1</RInstallPath>
```

Comment:
https://github.com/Open-Systems-Pharmacology/rSharp/issues/116

Reply:
I'm going to get rid of this dependency and fix it in the README before this is merged.